README.md

```diff
@@ -75,14 +75,14 @@ Install base packages:
 Create unprivileged user which will run the web applications:
 
     sudo adduser edxapp
-    sudo su edxapp
+    sudo -sHu edxapp
 
 Note that the `edxapp` user does not have sudo rights, so all following commands that are run with `sudo` will have to be run with a different user; for instance, the user with which you created the `edxapp` user. This is on purpose, as `edxapp` will be used to run the web services. But if you want to grant sudo rights to `edxapp`, run : `sudo usermod -a -G sudo edxapp`.
 
 Create folder in which everything will be installed:
 
-    sudo mkdir /opt/openedx
-    sudo chown edxapp:edxapp /opt/openedx
+    sudo mkdir /opt/Open edX
+    sudo chown edxapp:edxapp /opt/Open edX
```

Comment:
I'd rather we did not insert spaces in the install path. Spaces need to be escaped and they make install instructions more complex.